.github/actions/configure_dbt_environment/action.yaml

```diff
@@ -19,11 +29,7 @@ runs:
             echo "HEAD_REF=data-catalog";
           } >> "$GITHUB_ENV"
         else
-          echo "On pull request branch, setting dbt env to CI"
-          {
-            echo "TARGET=ci";
-            echo "CACHE_KEY=$GITHUB_HEAD_REF";
-            echo "HEAD_REF=$GITHUB_HEAD_REF"
-          } >> "$GITHUB_ENV"
+          echo "CI context did not match any of the expected environments"
+          exit 1
```

Comment:
Not fully intentional, but the build ended up testing this path when I accidentally referenced `GITHUB_REF_NAME` instead of `GITHUB_HEAD_REF` for this run: https://github.com/ccao-data/data-architecture/actions/runs/5801770404/job/15726741689